website/data/docs-remote-plugins.json

```
@@ -89,7 +89,8 @@
     "title": "HashiCups",
     "path": "hashicups",
     "repo": "hashicorp/packer-plugin-hashicups",
-    "version": "latest"
+    "version": "latest",
+    "isHcpPackerReady": true
```

Thanks for calling out this setting for demo purposes. We can go in and add the details for the 5 plugins that are HCP Packer registry: vsphere, googlecompute, amazon, docker, azure. 

```suggestion
    "isHcpPackerReady": false
```